db/migrate/20200809160820_create_tickets.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateTickets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tickets do |t|
+      t.references :contact, null: true, foreign_key: true
+      t.string :ticket_id
```

Comment:
also null: false. Not sure if unique: true is a thing, but it needs a unique enforcement

Reply:
Doesn't seem to be a way to make a non-index column here unique, but it's enforced in the ticket model